### PR TITLE
ASDK-1456:  Update readme with ChannelHashtags screenshots

### PR DESCRIPTION
**Pull request by OstapOpalynskyi-fw**
**What**
Update readme with ChannelHashtags screenshots

**Why**

ASDK-1456

**How**
Add screenshots and for the ChannelHashtags screen

**Result**

https://github.com/loopsocial/firework_android_sdk_examples/tree/ASDK-1456-update-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,41 +117,50 @@ You need a feed resource with a Multiple Host Livestream item.
 
 ## Shopping example app
 
-In this example app, the `FwVideoFeedView` is initialized and integrated with the shopping features of the SDK.
-The app has two screens which demonstrate how to setup shopping for "Shopping cart mode" and "Shop now mode".
+In this example app, the `FwVideoFeedView` is initialized and integrated with the shopping features
+of the SDK.
+The app has two screens which demonstrate how to setup shopping for "Shopping cart mode" and "Shop
+now mode".
```
Comment (mohsenoid) [inline README.md:123]: Unnecessary new lines

File: README.md [hunk 2]
```diff
@@ -110,41 +117,50 @@ You need a feed resource with a Multiple Host Livestream item.
...
 2. `ShareLinkActivity` - handles the deep link and opens the shared video.
 
 Also, this example demonstrates how to replace the base URL of the shared video with a custom one.
 
-**Note**: [Starting in Android 12 (API level 31)](https://developer.android.com/about/versions/12/behavior-changes-all#web-intent-resolution), a generic web intent resolves to an activity in your app only if your app is approved for the specific domain contained in that web intent. If your app isn't approved for the domain, the web intent resolves to the user's default browser app instead.
+**
+Note**: [Starting in Android 12 (API level 31)](https://developer.android.com/about/versions/12/behavior-changes-all#web-intent-resolution)
+, a generic web intent resolves to an activity in your app only if your app is approved for the
+specific domain contained in that web intent. If your app isn't approved for the domain, the web
+intent resolves to the user's default browser app instead.
```
Comment (mohsenoid) [inline README.md:153]: ?

**Review by mohsenoid — APPROVED** [review]
Besides unnecessary new lines, LGTM.
I am also wondering if shouldn't we add hashtags to the details section?
image

**Comment by OstapOpalynskyi-fw** [comment]
> Besides unnecessary new lines, LGTM. I am also wondering if shouldn't we add hashtags to the details section? image

Hashtags is specific to the screen and it is visible in the input field on top of the screen. So I would say no.

File: README.md (hunk 1 above)
Comment (OstapOpalynskyi-fw) [reply README.md]: fixed, thanks

**Pull request merged**